gravitee-apim-gateway/gravitee-apim-gateway-http/src/main/java/io/gravitee/gateway/jupiter/http/vertx/AbstractVertxServerRequest.java

```diff
@@ -99,13 +99,13 @@ public String pathInfo() {
     }
 
     @Override
-    public MutableHttpRequest pathInfo(final String pathInfo) {
+    public AbstractVertxServerRequest pathInfo(final String pathInfo) {
```

Comment:
```suggestion
    public MutableRequest pathInfo(final String pathInfo) {
```
I don't like to return Abstract classes and the interface here is enough. WDYT?

Reply:
I don't think it is necessary to move to `MutableRequest` as it is an implementation and all others components will access these methods through interfaces so they'll never see this abstract class.